.github/workflows/bridge-ui-production.yml

```diff
@@ -8,7 +10,7 @@ on:
       - "bridge-ui-*"
 
 jobs:
-  Deploy-Production:
```

Comment:
For deployments, maybe we should use something like the following as a pattern:

[name]_[platform]_[network]_[type]
name: bridge-ui
platform: vercel|gcp
network: mainnet | hekla | i176661 (176661 is an internal network id)
type: production | preview

@KorbinianK @cyberhorsey @xiaodino @2manslkh @bennettyong @bearni95

Reply:
@KorbinianK maybe you can help me improve bridge-ui related workflow first to show people an example.

Reply:
@kimo-ice your input is appreciated

Reply:
honestly no i dont think so, its too complex. for the monorepo i think we should have names using this tool [hawk](https://github.com/rawnly/hawk).

this would make things much more readable and approachable. we can use the "repo" scope as well for multi-package workflows. i'm happy to take this on / take over this PR if you guys like this approach.

this is not meant to be perfect right now, but i think this is alright, we don't have too many files. it is clearly very redundant, but we can fix that later on as a lower priority item using tools like [nx](https://nx.dev).

Reply:
I have made some improvements to the bridge-ui workflow [here](https://github.com/taikoxyz/taiko-mono/pull/16951). @KorbinianK @d1onys1us you may take a look.
In terms of naming convention, hawk is a really nice tool for more organized workflows, and it don't conflict with having a naming convention pattern as well, we can still name directories/files according to the agreed pattern.